### DISCUSSION TITLE
[잡담] 이번주 과제 어렵나용

### DISCUSSION THREAD

**Discussion by yoomambo** [4Moyede/HexatonClass-01#69] [잡담]
?.?....?.?

꺄륵

**Comment by lsj0822** [comment]
어렵네용... 과제3번에서 1,2번은 그래도 꾸역꾸역 풀었는데, 3번은 접근부터 어렵네요

**Reply by yoomambo** [reply]
3번이 흔히 말하는 자연어처리 예제문제 정도 될거에요~

3번하면 자연어처리 입문 개이득

**Comment by NA-DK** [comment]
후....3번만 4시간째...

**Comment by dlrudgnsdlek** [comment]
3번은 split, strip 쓰는거 같은데,, 빈도는 도저히 어떻게 해야할지 모르겠어여,,,...,,,..,,.
